shell/common/shell.h

```diff
@@ -432,6 +478,15 @@ class Shell final : public PlatformView::Delegate,
       const Shell::CreateCallback<PlatformView>& on_create_platform_view,
       const Shell::CreateCallback<Rasterizer>& on_create_rasterizer);
 
+  static std::future<std::unique_ptr<Shell::ShellHolder>> initShellEnv(
```

Comment:
Our C++ name convention (https://google.github.io/styleguide/cppguide.html) is `InitShellEnv` for methods.